### ISSUE TITLE
ssh remote port?

### ISSUE THREAD

**Issue by ghost** [ColdGrub1384/LibTerm#81]
How do I specify the ssh port? e.g.:
`ssh user@example.com -p 1234`
still tries to connect to port 22.
Also .ssh/config is not read.

**Comment by kafeg** [comment]
the same for me

**Comment by cbertelli** [comment]
Yes, it world ne useful for me se well. In general, I would expect a command like
**help ssh** or
**ssh help** or
**ssh —help**
to explain what to do, but maybe the files are missing.
